addons/godaw_toolkit/API/GoDAW.gd

```diff
@@ -3,8 +3,7 @@ extends Node
 var instruments = {}
 
 func register_instrument(name: String, instrument: PackedScene):
-	if instrument.instance() is Instrument:
-		self.instruments[name] = instrument
+	self.instruments[name] = instrument
```

Comment:
if we do it like this how will we know if instrument is Instrument?

Reply:
Yes that's to be decided, but we'd be making an instance of the instrument just to check what type it is, which is fairly inefficient

Reply:
fair xP